### PR TITLE
All local changes

### PR DESCRIPTION
**Pull request by v-zinchenko**
Dronezone final version
Safor webpage without last section and footer

### CODE REVIEW

File: safor/index.html [hunk 1]
```diff
@@ -3,25 +3,25 @@
...
             <a href="#">Contacts</a>
         </nav>
     </header>
     <section class="first-section">
-        <div class="text">
-            <h1 class="off">SAFOR</h1>
-            <h1>switching your <br id="switching-your-br">life to safety mode</h1>
-            <p>Safor is an intelligent personal guard platform that empowers individuals to enjoy active <br id="active-br">and safe lives, <br id="safe-lives-br">while their friends and family <br id="friend-and-family-br">can enjoy peace of mind.</p>
-        </div>
+        <ul class="text">
```
Comment (ghost) [inline safor/index.html]: Its should be a string `<h1>`, not a list

File: safor/index.html [hunk 2]
```diff
@@ -3,25 +3,25 @@
...
+            <li class="off"><h1>SAFOR</h1></li>
+            <li><h1>switching your life to safety mode</h1></li>
+            <li><p>Safor is an intelligent personal guard platform that empowers individuals to enjoy active and safe lives, while their friends and family can enjoy peace of mind.</p></li>
```
Comment (ghost) [inline safor/index.html]: Separate this one into another block.

File: safor/index.html [hunk 3]
```diff
@@ -79,29 +79,29 @@ <h1>WHY SAFOR?</h1>
                     <h2>FOR PEOPLE</h2>
             </div>
             <ul>
-                <li><img src="./images/section5/pentagon.svg" alt="pentagon"></li>
-                <li>Do not waste time to explain your location and nature of the threat, guard patrols get <br id="get-br">your alarm in 1 second and advances to you no matter whether you are in motion or not.</li>
-                <li><img src="./images/section5/star.svg" alt="star"></li>
-                <li>You will be foundregardless of whether you are in town or on a trip. <br id="trip-br">The service covers the entire territory of the country and allows you to specify <br id="specify-br">the exact location even when in a crowded place</li>
-                <li><img src="./images/section5/triangle.svg" alt="triangle"></li>
-                <li>And we will help in any situation. Only after coming to you the calling <br id="calling-br">will be closed by the patrol arrived, whose goal is to protect and help you.</li>
+                <li><img src="images/section5/pentagon.svg" alt="pentagon"></li>
```
Comment (ghost) [inline safor/index.html]: Image and text for this image - one item. Show nesting in tags, code will be more readable. Example :
```
<li>
 <img/>
 <p></p>
</li>
```

File: safor/index.html [hunk 4]
```diff
@@ -79,29 +79,29 @@ <h1>WHY SAFOR?</h1>
...
+                <li>Do not waste time to explain your location and nature of the threat, guard patrols get your alarm in 1 second and advances to you no matter whether you are in motion or not.</li>
+                <li><img src="images/section5/star.svg" alt="star"></li>
+                <li>You will be foundregardless of whether you are in town or on a trip. The service covers the entire territory of the country and allows you to specify the exact location even when in a crowded place</li>
+                <li><img src="images/section5/triangle.svg" alt="triangle"></li>
+                <li>And we will help in any situation. Only after coming to you the calling will be closed by the patrol arrived, whose goal is to protect and help you.</li>
             </ul>
         </div>
         <div class="black-rect">
             <h2>FOR BUSINESS</h2>
```
Comment (ghost) [inline safor/index.html]: Same as white block

File: safor/index.html [hunk 5]
```diff
@@ -79,29 +79,29 @@ <h1>WHY SAFOR?</h1>
...
+                <li><img src="images/section5/hexagon.svg" alt="hexagon"></li>
+                <li>In making decisions, the quality of the services provided and the distribution of funds. We use all available resources to achieve maximum results in our work and communication.</li>
             </ul>
         </div>
     </section>
     <section class="form">
         <form action="#">
-            <h1>HAVE QUESTIONS?<br>CONTACT US</h1>
+            <h1>HAVE QUESTIONS? CONTACT US</h1>
             <div class="first-row">
```
Comment (ghost) [inline safor/index.html]: This block can be not separated into two different blocks. You should place all the inputs in one block and set position using flex.

File: safor/index.html [hunk 6]
```diff
@@ -33,23 +33,23 @@ <h1>switching your <br id="switching-your-br">life to safety mode</h1>
...
+                <img src="images/section2/playbutton.svg" alt="playbutton">
                 <p>WATCH NOW</p>
             </div>
         </div>
-        <div class="text">
-            <p>The existing types of protection in today's society can not always be adapted to the needs of the modern customers.</p>
-            <p>The SAFOR system is flexible and adaptive, providing personal protection to you and your loved once  at any time of the day, wherever they are.</p>
-            <p>Regardless the situation, the system has unlimited possibilities to make your life free and safe.</p>
-        </div>
+        <ul class="text">
```
Comment (ghost) [inline safor/index.html:38]: Something went wrong here. Please fix positioning. Also nesting p tag into li makes no sense. You can just put text in li item and style it as p.

File: safor/index.html [hunk 7]
```diff
@@ -59,15 +59,15 @@ <h1>WHAT IS SAFOR?</h1>
...
             </div>
-            <img src="./images/section3/phone_1.png" alt="phone">
-            <img src="./images/section3/phone_2.png" alt="phone">
+            <img src="images/section3/phone_1.png" alt="phone">
+            <img src="images/section3/phone_2.png" alt="phone">
         </div>
     </section>
     <section class="forth-section">
         <div class="text">
                 <h1>WHY SAFOR?</h1>
```
Comment (ghost) [inline safor/index.html]: We have different font style in design for this title.

File: safor/index.html [hunk 8]
```diff
@@ -3,25 +3,25 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
     <link href="https://fonts.googleapis.com/css?family=Roboto:500,700&display=swap" rel="stylesheet">
     <title>SAFOR</title>
 </head>
 <body>
```
Comment (ghost) [inline safor/index.html:10]: Looks like website is broken on resolutions 1680 and 1440

File: safor/index.html [hunk 9]
```diff
@@ -79,29 +79,29 @@ <h1>WHY SAFOR?</h1>
...
+                <li>In making decisions, the quality of the services provided and the distribution of funds. We use all available resources to achieve maximum results in our work and communication.</li>
             </ul>
         </div>
     </section>
     <section class="form">
         <form action="#">
-            <h1>HAVE QUESTIONS?<br>CONTACT US</h1>
+            <h1>HAVE QUESTIONS? CONTACT US</h1>
             <div class="first-row">
                 <input type="text" class="name" placeholder="name">
```
Comment (ghost) [inline safor/index.html]: On resolution 759 placeholder letters does not fit.

**Review by ghost — CHANGES_REQUESTED** [review]

File: safor/index.html [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+            </div>
+            <input type="text" class="message" placeholder="message">
+            <input type="submit" class="submit" value="send">
+        </form>
+    </section>
+    <footer>
+        <img class="logo" src="images/footer/logo.svg" alt="logo">
+        <div class="second-third-elements">
+            <p>© SAFOR 2018 All Rights Resevered 85 Great Portland Street, First Floor London, W1W7LT, United Kingdom</p>
+            <ul class="links">
```
Comment (ghost) [inline safor/index.html:134]: social buttons in wrong position on 1024px resolution, sizes of icons looks incorrect

File: safor/index.html [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+    </section>
+    <section class="form">
+        <form action="#">
+            <h1>HAVE QUESTIONS? CONTACT US</h1>
+            <div class="first-row">
+                <input type="text" class="name" placeholder="name">
+                <input type="email" class="email" placeholder="email">
+            </div>
+            <input type="text" class="message" placeholder="message">
+            <input type="submit" class="submit" value="send">
```
Comment (ghost) [inline safor/index.html:127]: Please remove bottom border on button.

File: safor/index.html [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+            <a href="#">Contacts</a>
+        </nav>
+    </header>
+    <section class="first-section">
+        <ul class="text">
+            <li class="off"><h1>SAFOR</h1></li>
+            <li><h1>switching your life to safety mode</h1></li>
+            <li><p>Safor is an intelligent personal guard platform that empowers individuals to enjoy active and safe lives, while their friends and family can enjoy peace of mind.</p></li>
+        </ul>
+        <hr>
```
Comment (ghost) [inline safor/index.html:23]: Why there are only five lines visible instead of 7 ?

File: safor/index.html [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+        </div>
+    </section>
+    <section class="form">
+        <form action="#">
+            <h1>HAVE QUESTIONS? CONTACT US</h1>
+            <div class="first-row">
+                <input type="text" class="name" placeholder="name">
+                <input type="email" class="email" placeholder="email">
+            </div>
+            <input type="text" class="message" placeholder="message">
```
Comment (ghost) [inline safor/index.html:126]: As i can see, distance between placeholder and border is different on 375 resolution and less

**Review by ghost — CHANGES_REQUESTED** [review]

File: safor/index.html (hunk 11 above)
Comment (ghost) [reply safor/index.html:127]: Still issue

**Review by ghost — DISMISSED** [review]

**Review by volo-maslow — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**